Esterni/ManualeUtente/components/c2-Requisiti_minimi_sistema.tex

```diff
@@ -0,0 +1,66 @@
+\section{Requisiti minimi di sistema}
+
+\subsection{Requisiti hardware}
+
+Non vi sono particolari requisiti minimi per l'utilizzo del software. Vi è da notare comunque che più la cartella da sincronizzare è grande e più il programma richiederà maggiori risorse.
```

Comment:
cambierei
'Vi è da notare comunque che più la cartella da sincronizzare è grande e più il programma richiederà maggiori risorse.'
in
'Vi è da notare comunque che più la cartella da sincronizzare sarà grande maggiori saranno le risorse richieste dal programma'

Reply:
python 3.9 non supporta windows 7 quindi ci sono dei requisiti minimi

fonte https://www.python.org/downloads/release/python-390/

Reply:
Questi sono requisiti hardware, sul software è segnato windows 10 versione minima